tailwind.config.mjs

```diff
@@ -1,50 +1,54 @@
-/** @type {import('tailwindcss').Config} */
```

Comment:
We need to figure out why all these changes

Reply:
I think that is because the problem with all files, cause I only added the font family